frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -63,10 +60,10 @@ void main(List<String> args) async {
     expect(resp.int16List, Int16List.fromList(List.filled(n, 42)));
     expect(resp.uint32List, Uint32List.fromList(List.filled(n, 42)));
     expect(resp.int32List, Int32List.fromList(List.filled(n, 42)));
-    expect(resp.uint64List, Uint64List.fromList(List.filled(n, 42)));
-    expect(resp.int64List, Int64List.fromList(List.filled(n, 42)));
     expect(resp.float32List, Float32List.fromList(List.filled(n, 42)));
     expect(resp.float64List, Float64List.fromList(List.filled(n, 42)));
+    expect(resp.uint64List, Uint64List.fromList(List.filled(n, 42)));
+    expect(resp.int64List, Int64List.fromList(List.filled(n, 42)));
```

Comment:
nit: Maybe we should add a few tests, checking the *value range*. For example, for a i64 and a Int64List, respectively, we need to assert it can pass a big integer (say 50bit long), instead of passing only "42".